web/client/plugins/QueryPanel.jsx

```diff
@@ -435,6 +446,43 @@ class QueryPanel extends React.Component {
  *        },
  *        "customItemClassName": "customItemClassName"
  *    }
+ *
+ * @example
```

Comment:
In plugin description above specify that this plugin has by default 3 panels and it can be customized by plugin injection. Then specify the available targets.